specs/pvs/corla.pvs

```diff
@@ -269,14 +276,16 @@ BEGIN
 %%% voter, a candidate, both, or neither.
   candidate: TYPE FROM person
 
-%%% Elections focus on contests, each of which represents a set of
-%%% choices that voters can make. Making a legal choice ranges
+%%% Elections focus on contests, each of which represents a 
+%%% choice that voters can make. Making a legal choice ranges
 %%% enormously across the Earth, from marking a vote for a candidate
 %%% by filling in a single bubble to enumerating a total order on all
-%%% choices in a contest. Across a given ballot, each contest has a
+%%% choices in a contest. Within any single elction, each contest has a
 %%% unique name.
 % @todo kiniry Need to break the cycle between the contest type and
-% the ballots theory.
+% the ballots theory. sfsinger suggests: contests are grouped into elecitons,
+% not ballots. So define contests first. Then a ballot is created to 
+% represent a set of contests. 
```

Comment:
The work of the NIST/EAC Interoperability Working group, Election Modeling subgroup, might be useful. (Yeah, pretty late to bring this up, but I just noticed their recent progress, FWIW).  The seem to have a BPMN2 model, pictures, XPDL, xml, who-knows-what at https://github.com/usnistgov/ElectionModeling.  See also
http://collaborate.nist.gov/voting/bin/view/Voting/ElectionModeling
Also the UML for the https://github.com/usnistgov/CastVoteRecords work.